portfolio/src/main/webapp/gallery.html

```diff
@@ -0,0 +1,445 @@
+                  </figcaption>
+                </figure>
+
+                <!-- BOTW -->
+                <figure class="tm-gallery-item">
+                  <a href="https://open.spotify.com/album/5maeycU97NHBgwRr2h2A4O?si=HGwqkPnlTOG2ybdug3Ffjw"
+                    target="_blank">
+                    <img src="img/botw.jpg" alt="Born This Way Album" class="" />
+                  </a>
+                </figure>
+
+                <!-- Dog Fact -->
+                <figure class="effect-goliath tm-gallery-item">
+                  <img src="img/dogFact.jpeg" alt="I have a dog" class="" />
+                  <figcaption>
+                    <h2>
+                      I Have a Small
+                      <span>Daughter</span>
+                    </h2>
+                    <p>She's a 2 years old american bully</p>
+                  </figcaption>
+                </figure>
+
+                <!-- IGor-->
+                <figure class="tm-gallery-item">
+                  <a href="https://open.spotify.com/album/5zi7WsKlIiUXv09tbGLKsE?si=0llEppu4R86cVT695MHIWw"
+                    target="_blank">
+                    <img src="img/igor.jpg" alt="Born This Way Album" class="" />
+                  </a>
+                </figure>
+
+                <!-- Lemonade -->
+                <figure class="tm-gallery-item">
+                  <a href="https://open.spotify.com/album/7dK54iZuOxXFarGhXwEXfF?si=iGrUaW4UQXavDduw1PTZ8Q"
+                    target="_blank">
+                    <img src="img/lemonade.png" alt="Pure Heroine Album" class="" />
+                  </a>
+                </figure>
+
+                <!-- Fall Fact -->
+                <figure class="effect-goliath tm-gallery-item">
+                  <img src="img/fallFact.jpeg" alt="I love Fall" class="" />
+                  <figcaption>
+                    <h2>
+                      My Favorite Season is
+                      <span>Fall</span>
+                    </h2>
+                    <p>Cozy sweaters + Pumpkin Spice Latte</p>
+                  </figcaption>
+                </figure>
+
+                <!-- Norman Rockwell -->
+                <figure class="tm-gallery-item">
+                  <a href="https://open.spotify.com/album/5XpEKORZ4y6OrCZSKsi46A?si=mUlKVOL3TQSQcvlek9nJvw"
+                    target="_blank">
+                    <img src="img/norman.jpg" alt="Norman Rockwell Album" class="" />
+                  </a>
+                </figure>
+
+                <!-- Nephews Fact -->
+                <figure class="effect-goliath tm-gallery-item">
+                  <img src="img/nephewFact.jpeg" alt="I have 3 nephews" class="" />
+                  <figcaption>
+                    <h2>
+                      I Have Three
+                      <span>Nephews</span>
+                    </h2>
+                    <p>But I see them more as my brothers</p>
+                  </figcaption>
+                </figure>
```

Comment:
Optional: Notice that you have many entities with the same (alternating) structure here. Instead of using pure HTML, you can replace the long code with a JS script, possibly by also using some data structures like list/array